source/jsonnet/england-wales/lib/placeholders.libsonnet

```diff
@@ -95,53 +97,33 @@ local firstPersonNamePossessiveForList(listName) = {
       },
       transform: 'concatenate_list',
     },
-    {
-      transform: 'format_possessive',
-      arguments: {
-        string_to_format: {
-          source: 'previous_transform',
-        },
-      },
-    },
+    transforms.formatPossessive,
   ],
 };
 
+local personName(sameNameTransform='') = (
+  if sameNameTransform == 'contains_same_name_items' then
+    {
+      placeholder: 'person_name',
+      transforms: [transforms.containsSameNameItems, transforms.formatFirstPersonName],
+    }
+  else if sameNameTransform == 'list_has_same_name_items' then
+    {
+      placeholder: 'person_name',
+      transforms: [transforms.listHasSameNameItems, transforms.formatFirstPersonName],
+    }
+  else
+    {
+      placeholder: 'person_name',
+      transforms: [transforms.concatenateList],
+    }
+);
+
 {
-  personName: {
-    placeholder: 'person_name',
-    transforms: [{
-      transform: 'concatenate_list',
-      arguments: {
-        list_to_concatenate: {
-          source: 'answers',
-          identifier: ['first-name', 'last-name'],
-        },
-        delimiter: ' ',
-      },
-    }],
-  },
+  personName: personName,
   personNamePossessive: {
     placeholder: 'person_name_possessive',
-    transforms: [
-      {
-        transform: 'concatenate_list',
-        arguments: {
-          list_to_concatenate: {
-            source: 'answers',
-            identifier: ['first-name', 'last-name'],
-          },
-          delimiter: ' ',
-        },
-      },
-      {
-        transform: 'format_possessive',
-        arguments: {
-          string_to_format: {
-            source: 'previous_transform',
-          },
-        },
-      },
-    ],
+    transforms: [transforms.concatenateList, transforms.formatPossessive],
```

Comment:
Do we ever need to apply same name logic to possessive names?

Reply:
I couldn't see anywhere outside of relationships, which is handled in a different part